README.md

```diff
@@ -63,7 +63,7 @@ First download `clasp`:
 
 ```sh
 npm install -g @google/clasp
-# Alternatively: sudo npm i -g grpc @google/clasp --unsafe-perm
+# Alternatively: sudo npm install -g grpc @google/clasp --unsafe-perm
```

Comment:
@grant any context as for when/why one try to `sudo` and add a 3rd party `grpc` package?

Reply:
> @grant any context as for when/why one try to `sudo` and add a 3rd party `grpc` package?

I don't know. It may be an old workaround. We need someone to completely uninstall Node, all libraries, and install from scratch and verify it works.

Reply:
I have clasp running on MacOS and Windows on "clean" node instances without need of this `grpc`.
If it solved any issue with clasp, it is likely a side-effect it seems.

Reply:
I found original mention of `grpc` in #190 
From what I can gather, some of the dependencies at the time used grpc but could require you to manually update to latest version.

Currently grpc is no longer mentionned in `node_modules` nor `package-lock.json`

I suggest we wipe this outdated "fix" (cf. #441 )